frontend/device/devicelistener.lua

```diff
@@ -227,7 +232,9 @@ if Device:hasFrontlight() then
     end
 
     function DeviceListener:onShowFlDialog()
-        Device:showLightDialog()
+        if Device:showLightDialog() then
+            UIManager:broadcastEvent(Event:new("FrontlightStateChanged"))
```

Comment:
I don't like it. showLightDialog should broadcast events directly (as it does now). Is there a benefit of doing this way?

Reply:
As you wish.